exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,317 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In this case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
```

Comment:
I would prefer that the two sentences not be combined into one line, but not so much that it stops me from Approving (in the GitHub sense of that word)

Reply:
(by that, I mean that I would prefer it if the sentence that starts with "If there" would be on a different line than the sentence that ends with "single thread")